### ISSUE TITLE
Composer gets installed but doesn't work (behind a proxy)

### ISSUE THREAD

**Issue by fdbatista** [composer/composer#5655]
Hello.

I managed to install composer in my computer (which runs Windows 8.1) behind my company's proxy using cntlm, but I haven't been able to make it download some packages.

It works, for example, with monolog/monolog package, which is described for testing purposes in the official documentation, but when I try to install laravel I get this:

![sin titulo](https://cloud.githubusercontent.com/assets/11459171/18324542/43f2f0e6-7567-11e6-8db2-ba66c30e35c3.png)

Basically, it returns this Composer\Downloader\TransportException: The Uniform Resource Locator (URL) does not use a recognized protocol.

Any idea or suggestion?
Thanks in advance.

**Comment by Seldaek** [comment]
Try to force https for all resources by running `composer config -g repo.packagist composer https://packagist.org` - that usually helps get around proxy issues.

**Comment by amad4biz** [comment]
I have the same issue with the above person. Can you please assist. I managed to install composer but since i am behind a proxy, I am not able to download any package. any help help will be much appreciated.
Where will run this command, i try but windows sating 'config ' is not an internal or external command

**Comment by Seldaek** [comment]
Closing issue as it has been quiet for a long time and there is no clear next step we can take.
